<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarInsumo</class>
 <widget class="QDialog" name="RegistrarInsumo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>167</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripci√≥n:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>71</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegistrar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>119</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio Unitario:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinPrecio">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>53</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>19</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinCantidad">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>37</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDescripcion">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinCantidad</tabstop>
  <tabstop>txtNombre</tabstop>
  <tabstop>spinPrecio</tabstop>
  <tabstop>btnRegistrar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
